src/wazuh_db/wdb_agent.c

```diff
@@ -984,12 +973,23 @@ char *get_agent_date_added(int agent_id){
                 *endl = '\0';
             }
 
+            if (sscanf(date, "%d-%d-%d %d:%d:%d",&t.tm_year, &t.tm_mon, &t.tm_mday, &t.tm_hour, &t.tm_min, &t.tm_sec)<6) {
+                merror("Date not read for agent %d", agent_id);
```

Comment:
Change this error message: `Invalid date format in file '%s' for agent '%d', TIMESTAMP_FILE, agent_id`

Reply:
```suggestion
                merror("Invalid date format in file '%s' for agent '%d'",TIMESTAMP_FILE, agent_id);
```